[DEVR-113] Global Accounts API Integration

Asfiroth opened the pull request:
- Enable create turnkey accounts
- Enable create zerodev smart contracts

YokySantiago commented on src/middleware.ts:
```diff
@@ -71,16 +87,12 @@ export const middleware = async (
         return authMiddleware(request, event);
       }
 
+      console.info('got here');
```
Let's remove this console log

YokySantiago commented on src/services/globalAccount.ts:
```diff
@@ -0,0 +1,159 @@
+import { turnkeyApiClient, turnkeyConfig } from '@/config/turnkey';
+
+// configurations
+const PROJECT_ID = 'c3526d90-4977-44e3-8584-8820693a7fd9';
```
Let's add this one to the env variables

YokySantiago reviewed (changes requested):
I left some small comments but in general LGTM. Is this deployed in the staging environment?

Asfiroth commented:
> I left some small comments but in general LGTM. Is this deployed in the staging environment?

Hi @YokySantiago addressed the comments, and wanted to ask you about staging: why are staging deployments disabled?
